test/manifest.ts

```diff
@@ -47,13 +51,25 @@ describe('Manifest', () => {
       manifest,
       lastReleaseSha,
       mergedPRFiles,
+      fixtureFiles,
+      inlineFiles,
       addLabel,
     } = options;
-    if (manifest !== undefined && lastReleaseSha) {
-      sandbox
-        .stub(github, 'getFileContentsWithSimpleAPI')
-        .withArgs('.release-please-manifest.json', lastReleaseSha, false)
+    const mock = sandbox.mock(github);
+    if (manifest) {
+      mock
+        .expects('getFileContentsWithSimpleAPI')
+        .once()
+        .withExactArgs('.release-please-manifest.json', lastReleaseSha, false)
         .resolves(buildGitHubFileRaw(manifest));
+    } else if (manifest === false) {
+      mock
+        .expects('getFileContentsWithSimpleAPI')
+        .once()
+        .withExactArgs('.release-please-manifest.json', lastReleaseSha, false)
+        .rejects(Object.assign(Error('not found'), {status: 404}));
+    } else {
+      mock.expects('getFileContentsWithSimpleAPI').never();
```

Comment:
really appreciating the strictness that `mock` allows - all though I do wonder if I'm testing the implementation details too closely...